### PR TITLE
Release 1.7.0

### PR DESCRIPTION
**Pull request by florianschiller**
## Release 1.7.0

* Integration of version Gematik Authenticator version 4.6.0
* Finalization of the new single HBA and SMC-B authentication flows
* Finalization of the new multi authentication flows

## Release 1.6.0

* Dependency updates

## Release 1.5.0

* Introduced new single HBA and SMC-B authentication flows
* Added new HBA and SMC-B identity providers to the sample realm
* Status endpoint renamed (from `authenticationStatus` to `status`)
* Next-Step endpoint renamed (from `authenticatorNextStep` to `nextStep`)

## Release 1.4.0

* Support for the new multi authentication flow added

## Release 1.3.0

* The Authenticator version is now appended to the MDC and the auth session notes
* Update to Keycloak 21.1.2

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ support the new theme with this plugin.
...
 | Authenticator Url                    | http://localhost:8000 <br/> http://localhost:39000 <br/> authenticator://                                                                           | Url to open the Authenticator app <br/> if `authenticator://`, Authenticator app is started, if it has not already been started and response is opened in a new tab <br/> if `http://localhost:8000` or `http://localhost:39000` Authenticator app needs to be running, response is opened in same tab |
+| Authentication Flow                  | Either single, HBA, SMCB or multiple request                                                                                                        | Choose your preferred authentication flow. When single, HBA or SMCB is selected, only a single request is made to the Gematik-Authenticator, but this flow requires an Authenticator in version 4.6.0. Otherwise, multiple requests are made to the Gematik-Authenticator.                             |
```
Comment (DaMo86) [inline Readme.md]: at least version / version 4.6.0 or higher

File: docker-compose.yml [hunk 2]
```diff
@@ -45,12 +45,12 @@ services:
       JAVA_OPTS: -Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005
     volumes:
       - ./sample-realm:/opt/keycloak/data/import/
-      - ./target/gematik-idp-1.2.0.jar:/opt/keycloak/providers/gematik-idp-1.2.0.jar
+      - ./target/gematik-idp-1.7.0-SNAPSHOT.jar:/opt/keycloak/providers/gematik-idp-1.7.0-SNAPSHOT.jar
```
Comment (DaMo86) [inline docker-compose.yml:48]: do we need to support SNAPSHOT versions here?

File: docs/puml/new-authentication-flow.puml [hunk 3]
```diff
@@ -0,0 +1,90 @@
+@startuml ogr
```
Comment (DaMo86) [inline docs/puml/new-authentication-flow.puml]: I guess we should use another name here?

**Review by DaMo86 — APPROVED** [review]
only thing that we could improve are the OGR reference in the flow diagrams, but this is no blocker from my pot

File: Readme.md (hunk 1 above)
Comment (DaMo86) [reply Readme.md]: updated it myself

File: docs/puml/new-authentication-flow.puml (hunk 3 above)
Comment (florianschiller) [reply docs/puml/new-authentication-flow.puml]: I exchanged OGR with APP everywhere. 👍

**Pull request merged**